keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/Hamiltonian.java

```diff
@@ -80,7 +80,7 @@ public NetworkSamples getPosteriorSamples(final BayesianNetwork bayesNet,
     public NetworkSamplesGenerator generatePosteriorSamples(final BayesianNetwork bayesNet,
                                                             final List<? extends Vertex> fromVertices) {
 
-        return new NetworkSamplesGenerator(setupSampler(bayesNet, fromVertices), ProgressBar::new);
+        return new NetworkSamplesGenerator(setupSampler(bayesNet, fromVertices), StatusBar::new);
```

Comment:
If we're generating Samples - do we really want to display any details on sampling?  Seems like the program itself could measure at that point and also stop us doing anything useful between samples...?

Reply:
I still think there is value in showing information about sampling. If someone is streaming samples, it would be useful for them to still know it is making progress. If they did want to use their own measurements, they could disable the status bar.